src/components/controls/controlHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { HeaderContainer } from "./styles";
+import { AnnotatedTitle } from "./annotatedTitle";
+
+type Props = {
+  title: string
+  tooltip: JSX.Element
+  mobile: boolean
+}
+
```

Comment:
[Force-pushed](https://github.com/nextstrain/auspice/compare/c9e44807721923586f25c182b50d3fef83e3032a..b3bceb5a7d2a672bea05e7e4f8320122280ca01c) all docstring updates.

I agree to not detail arguments/types in component docstrings, but I think it's still valuable to have a description in the prop type definition (which I've added in the force-push). This involved a lot of copy/pasting due to the prop drilling that's going on in the current implementation, and is prone to getting out of sync over time. Maybe using [contexts](https://react.dev/learn/passing-data-deeply-with-context) is a better approach here.

Reply:
That's a good suggestion - applied in [force-push](https://github.com/nextstrain/auspice/compare/1864c9cf1a83f655b87cbd101403c98e2ff10c25..ea97343643bc052a90f667025c4fa76b74d7cbd3). It's definitely much more maintainable.

Reply:
The way props are typed really shouldn't block this PR, but I'd like to take this opportunity to think about the options more since this is the first extended usage of TypeScript in this codebase.

---

[Here's](https://github.com/nextstrain/auspice/compare/ea97343643bc052a90f667025c4fa76b74d7cbd3..369425a3a5f895c2f3cd2ab4d14896d528db7d65) another option which exports prop types in its entirety, which is useful for cases where a group of props is often passed around together. This PR seems like an ideal place to consider it given many scenarios are covered:

1. [[ex](https://github.com/nextstrain/auspice/blob/369425a3a5f895c2f3cd2ab4d14896d528db7d65/src/components/controls/controlHeader.tsx#L5)] All props of a children are props of the component, and the names are the same.
2. [[ex](https://github.com/nextstrain/auspice/blob/369425a3a5f895c2f3cd2ab4d14896d528db7d65/src/components/controls/panelHeader.tsx#L8-L9)] All props of a children are props of the component, but some have different names. This requires using [indexed access types](https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html).
3. [[ex](https://github.com/nextstrain/auspice/blob/369425a3a5f895c2f3cd2ab4d14896d528db7d65/src/components/controls/panelSection.tsx#L10)] Some props of a children are props of the component. This requires indexed access types even if name is the same.
4. [[ex](https://github.com/nextstrain/auspice/blob/369425a3a5f895c2f3cd2ab4d14896d528db7d65/src/components/controls/panelSection.tsx#L12-L13)] The component has its own types.
5. [[ex](https://github.com/nextstrain/auspice/blob/369425a3a5f895c2f3cd2ab4d14896d528db7d65/src/components/controls/panelHeader.tsx#L7-L19)] A mix of everything.

This has even less duplication and (to me) is more readable when aliases such as `AnnotatedTitleProps` are used, but I can see how it adds complexity.

@jameshadfield what do you think?

Reply:
I've not written enough TS + React code to make a decision, but I'd like to spend some time (here & there) looking at what others have done / reading things on this subject. 

For the purposes of this PR can you pick either of your implementations, and then break this out into a separate issue that can bubble along in the background for a while?

Reply:
Yes, for sure. I'll stick your suggestion (what's currently included in this PR).

Reply:
#1723